src/library_webgpu.js

```diff
@@ -1556,7 +1560,11 @@ var LibraryWebGPU = {
     var writeSize = WebGPU.makeExtent3D(writeSizePtr);
     // This subarray isn't strictly necessary, but helps work around an issue
     // where Chromium makes a copy of the entire heap. crbug.com/1134457
```

Comment:
Might be worth updating the comment, or adding to it.   Seems like the rational for the `subarray` call and the `slice` call are different/unrelated?  

Perhaps also rename the local variable to be more general.  Perhaps `dataview`?

Reply:
Yup good call, I can do that - but before that I wasn't sure what the intended code should be assuming the Chrome bug is fixed, which it very well might be. I don't know enough about the original problem to say for sure.

For ref original submits:
https://github.com/emscripten-core/emscripten/commit/4b5c4f0694174e081661944ab3ffdb43dd1949b8
https://github.com/emscripten-core/emscripten/commit/37941c7f158cd00c171aaac5c6b4a749b4db3587

&

Reply:
That chrome bug was indeed fixed.. looks like over 1.5 years ago: https://chromium.googlesource.com/chromium/src/+/1cb27e77a57bb53069c2df348ea8a713dd3bff1c

Given that,  I think its fine to remove the workaround for that bug.

Reply:
We can definitely remove the workaround for bug 1134457. I'm not sure about bug 1201109 either unfortunately. If we can spot check it and make sure it works, then I think we can remove the workaround.

But it would be good to test this in a WebGPU conformance test so we can be sure it keeps working in the future. Filed https://github.com/gpuweb/cts/issues/3241